hphp/test/frameworks/frameworks.yaml

```diff
@@ -632,6 +632,36 @@
     clowns:
 	  # Dependent on system language settings.
       - twig/test/Twig/Tests/CompilerTest.php
+  typo3:
+    url: git://github.com/TYPO3/TYPO3.CMS.git
+    branch: master
+    commit: 42ccc0ce1850f3b2f9c7a01efe49230687bf579a
```

Comment:
if you're not using the latest tagged release, you need to include a comment explaining why